app/stats/forms.py

```diff
@@ -0,0 +1,26 @@
+from django import forms
+
+TIME_SCOPES = [
```

Comment:
For these, you might consider using the newer `choices` in Django. They help because they allow you more freedom to change that the string representation of information is, without needing to change the code in loads of places. As ever Adam Johnson has a helpful blogpost:

https://adamj.eu/tech/2020/01/27/moving-to-django-3-field-choices-enumeration-types/

https://docs.djangoproject.com/en/4.0/ref/models/fields/#enumeration-types